_content/articles/the-sal-an-alternative-buddhist-holy-tree_ireland-John-D.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Sal — An Alternative Buddhist Holy Tree?"
+authors:
+  - "John D. Ireland"
+external_url: "https://journal.equinoxpub.com/BSR/article/view/14508/16725"
+source_url: " https://doi.org/10.1558/bsrv.v17i1.14508"
+drive_links:
+  - "https://drive.google.com/drive/folders/10FhXe3dtyCmt_uQt0VXD8hpMJaDCjS2T"
+course: nature
+tags:
+  - trees
+  - Nepal
+  - India
+year: 2000
+month: June
+journal: Buddhist Studies Review
+volume: 17
+series: 
+number: 1
+pages: "1--2"
+---
```

Comment:
I liked your description, but it should be _in_ the file (not the PR, which is just a message to me), below this `---`  And the pull quote needs a `> ` at the beginning to mark it as a quote. Also, feel free to remove unnecessary parentheticals from pull quotes if you like. If you keep this one, please fix it. The automatic OCR mangled some of the characters.